tests/opentracer/test_tracer_tornado.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from nose.tools import eq_, ok_
+
+from opentracing.ext.scope_manager.tornado import TornadoScopeManager
+from tests.opentracer.test_tracer import get_dummy_ot_tracer
+
+
+def get_dummy_tornado_tracer():
+    return get_dummy_ot_tracer('tornado_svc', {}, TornadoScopeManager())
+
+
+@pytest.fixture()
+def nop_tracer():
+    return get_dummy_tornado_tracer()
+
+
+class TestTracerTornado():
+
+    def test_sanity(self, nop_tracer):
```

Comment:
I don't think it tests Tornado. Here you're testing the `TornadoScopeManager` but in a sync context. You should spin-up a Tornado application, do some context switches (`yield`-ing the execution) and see that the parenting is correct.

I'd say to move this part out of this PR. The PR was about fixing our test suite and not adding more tests. We'll be sure that our scope managers implementation are properly tested.

Reply:
Yes this was just meant to be a minimal basic test since the references to such a test were included in `tox.ini` (here: https://github.com/DataDog/dd-trace-py/blob/opentracer/tox.ini#L255).

So I think it makes sense to include this test in this PR.

As for testing the actual scope manager logic, I think this should be left for the `opentracing` library to do. We should just check that the scope manager exists and functions.

Reply:
Gotcha. If that's the only scope of the test, let's add a comment / docstring explaining why we're not testing the full behavior (so that it's already covered in the OT module)